<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="WAEAEDXB01-0535" name="com.Edenred.testCases.MoneyTransfer.Negative_Tests.TC_02_Get_Beneficiary" tests="3" failures="0" timestamp="14 Jun 2022 12:14:07 GMT" time="17.850" errors="2">
  <testcase name="TC_01_Get_Beneficiary" time="7.386" classname="com.Edenred.testCases.MoneyTransfer.Negative_Tests.TC_02_Get_Beneficiary"/>
  <testcase name="TC_02_FXRATE_Invalid_USERID" time="5.200" classname="com.Edenred.testCases.MoneyTransfer.Negative_Tests.TC_02_Get_Beneficiary">
    <error type="java.lang.ClassCastException" message="com.aventstack.extentreports.markuputils.CodeBlock cannot be cast to java.lang.Throwable">
      <![CDATA[java.lang.ClassCastException: com.aventstack.extentreports.markuputils.CodeBlock cannot be cast to java.lang.Throwable
at com.Edenred.testCases.MoneyTransfer.Negative_Tests.TC_02_Get_Beneficiary.TC_02_FXRATE_Invalid_USERID(TC_02_Get_Beneficiary.java:93)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC_02_FXRATE_Invalid_USERID -->
  <testcase name="TC_03_FXRATE_AmountNotExists" time="5.264" classname="com.Edenred.testCases.MoneyTransfer.Negative_Tests.TC_02_Get_Beneficiary">
    <error type="java.lang.ClassCastException" message="com.aventstack.extentreports.markuputils.CodeBlock cannot be cast to java.lang.Throwable">
      <![CDATA[java.lang.ClassCastException: com.aventstack.extentreports.markuputils.CodeBlock cannot be cast to java.lang.Throwable
at com.Edenred.testCases.MoneyTransfer.Negative_Tests.TC_02_Get_Beneficiary.TC_03_FXRATE_AmountNotExists(TC_02_Get_Beneficiary.java:120)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TC_03_FXRATE_AmountNotExists -->
</testsuite> <!-- com.Edenred.testCases.MoneyTransfer.Negative_Tests.TC_02_Get_Beneficiary -->
